components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemOffer.vue

```diff
@@ -134,7 +134,7 @@ const insufficientBalance = computed(
 
 const offerPriceInvalid = computed(() => {
   if (offerPrice.value) {
-    return Math.sign(offerPrice.value) !== 1
+    return offerPrice.value < 0.01
```

Comment:
I also wonder why it should be 0.01 instead of the other limitation.

Reply:
there's a limit on basilisk which doesn't accept any offer below 0.01, no idea why, but 0.01 works and 0.009 doesn't. the issue itself reports an error which occurs if the price is < 0.01

Reply:
I would put this number in constants.ts

Reply:
> I would put this number in constants.ts and change `min` value in input `offerPrice`

done